Internal Server Error: /api/match
Traceback (most recent call last):
  File "C:\Python312\Lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\django\utils\deprecation.py", line 131, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\django\middleware\common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\django\middleware\common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 127.0.0.1:8000/api/match/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
Failed to upload to tmpfiles.org. Status code: 500
Internal Server Error: /api/match/
Failed to upload to tmpfiles.org. Status code: 500
Internal Server Error: /api/match/
Failed to upload to tmpfiles.org. Status code: 500
Internal Server Error: /api/match/
Failed to upload to tmpfiles.org. Status code: 500
Internal Server Error: /api/match/
Failed to upload to tmpfiles.org. Status code: 500
Internal Server Error: /api/match/
Failed to upload to tmpfiles.org. Status code: 500
Internal Server Error: /api/match/
Internal Server Error: /api/match/
